pkg/core/player/player.go

```diff
@@ -311,22 +305,23 @@ const (
 	AirborneXiao
 	AirborneVenti
 	AirborneKazuha
```

Comment:
won't be used i believe

Reply:
pass for now. maybe someone will do in future